### ISSUE TITLE
This channel is not responding when user clicks "yes" to add request...

### ISSUE THREAD

**Issue by KnightCat83** [ngovil21/PRC.bundle#75]
I'm just getting around to implementing RequestChannel rather than manage all of this myself through a Facebook group. I'm planning to start using CouchPotato but, I was hoping to just capture requests via the channel for now. Do I need to have everything else setup? When someone clicks "yes" they just get the "this channel is not responding...

Any advice or suggestions are much appreciated.

**Comment by Kyrluckechuck** [comment]
Just following up on this -- Any status or other info that could be provided to help?

**Comment by ngovil21** [comment]
Yes which client is this happening on and if you could provide a log when the user has tried to add the request to see if there is an error. You can email it to me at ngovil21@gmail.com.

**Comment by Kyrluckechuck** [comment]
Hey @ngovil21 ,

The client is a basic web-browser, in this case Firefox 61.0.1 (64-bit) [Windows 10 Pro 64-bit, Build 17134]
Server is running Ubuntu 18.04 LTS with PMS [1.13.4.5251-2e6e8f841] installed via [plexupdate](https://github.com/mrworf/plexupdate)

Attached is a *roughly* sanitized error log, I can email it as well if that is better, just let me know!

[plex_request_error.log](https://github.com/ngovil21/RequestChannel.bundle/files/2201874/plex_request_error.log)
